services/retrieve_and_rank/v1.js

```diff
@@ -246,6 +288,36 @@ module.exports = function (RED) {
       });
     },interval);
   }
-};
 
 
+    function checkClusterStatus(retrieve_and_rank,msg,node,cluster_id) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['checkClusterStatus' is defined but never used.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2404865885/issues/source?bid=3225115&fileBranchId=3233374#l293)